test/templates/test/auto_complete_test.js

```diff
@@ -0,0 +1,64 @@
+suite('Autocomplete', function() {
```

Comment:
Can we make it in the same style as the other tests? I don't prefer BDD myself, but since we went that route already...